lib/fake-xhr/index.test.js

```diff
@@ -1995,6 +1995,22 @@ describe("FakeXMLHttpRequest", function () {
                     FakeXMLHttpRequest.useFilters = false;
                 }
             });
+
+        it("correctly calls events on the fakeXhr",
```

Comment:
It's often difficult to understand what "correct" means in a test's expectation without reading the test implementation. If there's mistake in the implementation, then the reader will have an incorrect understanding of what the expectation is.

Would you mind updating the text of the expectation to be specific about what is expected?

Reply:
Yeah thats no problem at all. Did you have something in mind?

Reply:
Maybe just remove "correctly" ... then it just reads "calls events on the fakeXhr" ... that's pretty easy to understand.